trussworks/react-uswds
# chore: add storybook theme

# Summary

## Related Issues or PRs

None, internal Truss design-reserve [task](https://trello.com/c/lWVWhWdL/21-figure-out-branding-for-the-react-uswds-storybook)

## How To Test

- Check storybook colors & font for adherence to USWDS theme
- Read and click links on welcome page
- Check that panel shows only on components (hidden, as it's blank, on welcome, swatch, and template pages)

### Screenshots (optional)
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/764090/234693657-1b7753a6-f6b1-4ea6-b6c0-555729f3d5fb.png">
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/764090/234693713-c9af6b59-213e-4871-8ddb-bcb66fe76609.png">
